sci-mathematics/drgeo/drgeo-1.1.0-r1.ebuild

```diff
@@ -18,39 +16,35 @@ SRC_URI="
 
 SLOT="0"
 LICENSE="GPL-2"
-KEYWORDS="amd64 ~ppc x86"
+KEYWORDS="~amd64 ~ppc ~x86"
 IUSE="nls"
 
 RDEPEND="
-	x11-libs/gtk+:2
+	x11-libs/gtk+:2=
```

Comment:
you sure about this? GTK+ is usually ABI stable for major SLOTs. And if it wasn't, the GNOME guys will likely not introduce subslots for GTK, so it's pointless either way.